mmvb_backend/case_synthesizer/api/views.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework.response import Response
+from rest_framework.status import HTTP_201_CREATED, HTTP_400_BAD_REQUEST
+from rest_framework.viewsets import GenericViewSet
+
+from case_synthesizer.api.schemas import CaseSynthesizerSchema
+from case_synthesizer.api.serializers import CaseSynthesizerSerializer
+from case_synthesizer.generator import generate_cases
+from cases.api.serializers import CaseSerializer
+
+
+class CaseSynthesizerViewSet(GenericViewSet):
+    schema = CaseSynthesizerSchema(
+        tags=["Case Synthesizer"], operation_id_base="synthesizeCases"
+    )
+    serializer_class = CaseSynthesizerSerializer
+
+    def get_response_serializer(self, *args, **kwargs):
+        serializer_class = CaseSerializer
+        kwargs["context"] = self.get_serializer_context()
+        return serializer_class(*args, **kwargs)
+
+    def create(self, request):
+        serializer = self.serializer_class(data=request.data)
+        if serializer.is_valid():
+            quantity = serializer.validated_data.get("quantity")
```

Comment:
Not sure if this is happening somewhere else, but I'd enforce `quantity` (this is the number of cases, right?) to be `0 < quantity <= 200`

**Bonus** making sure it's self-documenting

Reply:
Good catch, I've just implemented a validator for that.
I'm not sure how to make it self-documented on the openapi autogenerated page though.
Maybe we can create an issue for further investigating it